### ISSUE TITLE
UI: Inplace Rename

### ISSUE THREAD

**Issue by warpkanal** [marta-file-manager/marta-issues#176]
Renaming files is something I do very often. I find it more nice and fast if a rename operation is inplace and not with an extra dialog (esp since the dialog has no additional options). Actually in the very same way that Finder and many others do it :)

**Comment by nanoant** [comment]
Yes, please in place rename with active arrows like in Total Commander please.

So, when I activate rename and tap up/down arrow I get prev/next file rename.

**Comment by eugenesvk** [comment]
Agree with the ↓↑ arrows, but the ←→ arrows (with various modifiers) should navigate text to make renames easier, not preform other Marta actions

**Comment by hanoii** [comment]
I also want something like this, I am actually used to the click wait click method of finder, so I would like if that is supported besides some quicker keyboard way, arrows could perfectly be one.

Note: Using EAP 3 :D
